website/contribute/developer-setup.md

```diff
@@ -139,7 +139,10 @@ and more importantly also try to improve the process along the way as well.
  - Sending a Pull Request
    - Format commit and the pull request title like `[HUDI-XXX] Fixes bug in Spark Datasource`, 
      where you replace `HUDI-XXX` with the appropriate JIRA issue. 
+   - Pull request titles must have either `[HUDI-XXX]` or `[MINOR]` in their title. Note the brackets and capitalization.
    - Please ensure your commit message body is descriptive of the change. Bulleted summary would be appreciated.
+   - You must follow the instructions in the template and fill out all fields to pass our compliance checks.
+   - Do not remove or modify any headings in the template.
```

Comment:
We allow `Risk level` right?

Reply:
You can get rid of (write none, low medium or high below) but you still need the Risk Level heading, and you could remove documentation update entirely and validation will still pass. But I think the instruction is easier to follow if we just say don't modify any of them